lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -1079,11 +1079,10 @@ mkCheckpointEntity wid wal =
         , checkpointBlockHeight = bh
         , checkpointGenesisHash = BlockId (coerce (gp ^. #getGenesisBlockHash))
         , checkpointGenesisStart = coerce (gp ^. #getGenesisBlockDate)
-        , checkpointSlotLength = coerceSlotLength $ gp ^. #getSlotLength
-        , checkpointEpochLength = coerce (gp ^. #getEpochLength)
+        , checkpointSlotLength = 0
+        , checkpointEpochLength = 0
         , checkpointEpochStability = coerce (gp ^. #getEpochStability)
-        , checkpointActiveSlotCoeff =
-            W.unActiveSlotCoefficient (gp ^. #getActiveSlotCoefficient)
+        , checkpointActiveSlotCoeff = 0
```

Comment:
What about renaming the fields with a `Unused` suffix so it is a bit more transparent / consistent with other unused fields? It's a bit sad that we can't drop columns in SQLite tables :(

Reply:
OK done.